pkg/logs/build_logs.go

```diff
@@ -122,24 +122,20 @@ func (c *BuildLogsClient) watchReadyContainers(ctx context.Context, readyContain
 
 			switch r.Type {
 			case watch.Added, watch.Modified:
-
-				for _, c := range pod.Status.InitContainerStatuses {
-					if c.State.Waiting == nil {
-						readyContainers <- readyContainer{
+				containers := map[string]readyContainer{}
+				for _, c := range append(pod.Status.InitContainerStatuses, pod.Status.ContainerStatuses...) {
```

Comment:
what is the behavior of logs when they are run in regular containers? is the behavior of this package identical?

also, I'm surprised there were no test changes

Reply:
the only change is that we make sure the containers are pushed to the channel jn the order they are defined in the spec. The behavior shouldn't change for users, but I will see if i can add a test